lnrpc/rpc.proto

```diff
@@ -1704,7 +1704,17 @@ message EdgeLocator {
 }
 
 message QueryRoutesResponse {
+    /**
+    The route that results from the path finding operation. This is still a
+    routed failed to retain backwards compatibility.
+    */
     repeated Route routes = 1 [json_name = "routes"];
+
+    /**
+    The success probability of the returned route based on the current mission
+    control state. [EXPERIMENTAL]
```

Comment:
I think we need to introduce something like this. At some point, the main rpc server will be gone and how to indicate experimental fields then? I also don't think we want to make copies of rpc calls in sub servers just to be able to add a single experimental field. There are more experimental fields coming up. For invoice htlcs in #3390 and channel metrics in #3332.

Reply:
What's the rationale for having this field be experimental then? We should also come up with a standard for marking these as experimental since #3332 has a different format.

Reply:
It is not very critical data, mainly meant for debugging. If we mark it as experimental, we can change the field without deprecating it in a major release first. It is primarily signaling what users can expect of this field in terms of api stability.

But this is all open for discussion, just as the format is. 

I also looked to see if we could add something more structured like you can with `deprecated`, but it doesn't seem to be easy in proto.